CONTRIBUTING.md

```diff
@@ -63,6 +63,20 @@ To run unit tests or Web Platform Tests against a pull request, add one or more
 | `T-macos` | Unit tests: macOS |
 | `T-windows` | Unit tests: Windows |
 
+## AI contributions
+
+Contributions must not include content generated by large language models or other probabilistic tools, including but not limited to Copilot or ChatGPT. This policy covers code, documentation, pull requests, issues, comments, and any other contributions to the Servo project.
```

Comment:
I would argue that using large language models, by definition, is a failure to make reasonable efforts to check the correctness of our contributions. When we write code, for example, we act as the author. We are in the driver’s seat, deciding how to solve problems and reason about the correctness of the compiler and its output.

This is not the case for the output of generative AI tools, where we act as something more like a reviewer. Being in the passenger seat is always going to put us at a disadvantage, because we have to fight the incentive to accept the output as correct in order to think critically about it.